lsusb.c

```diff
@@ -288,6 +288,32 @@ static void dump_device(
 		read_sysfs_prop(serial, sizeof(serial), sysfs_name, "serial");
 	}
 
+    int speed = libusb_get_device_speed(dev);
```

Comment:
This should go up to the top of the function where the other variables are defined.

Reply:
Done

Reply:
Done